docs/security/workbenchAssets.md

```diff
@@ -0,0 +1,92 @@
+# Workbench Assets
+
+Uberfire implements access control mechanisms for most of its workbench assets. Read only permissions are supported for menu items, perspectives, screens, editors, splash screens and popup screens. That means, it is possible to add new permission entries into the authorization policy in case protected access is required for any of those asset types.
+
+Imagine, for instance, we are developing a workbench screen annotated as  `@WorkbenchPerspective(identifier = "HomeForManagers")`.  By default, assets are not protected unless permissions are added to the security policy. So, imagine now, we want to grant access only to individual perspectives:
+
+```
+# Manager role only have access to the HomeForManagers perspective
+role.manager.permission.perspective.read=false
```

Comment:
@dgutierr why I need to set this to false? If "assets are not protected unless permissions are added" I would presume that if I set a permission for managers, all the other roles are false by default.